### ISSUE TITLE
Category chart with only one x-axis point collapses when animations are enabled

### ISSUE THREAD

**Issue by codinronan** [amcharts/amcharts4#215]
Hello,

This is an Angular 6 application.

I'm not sure if I am doing something wrong or if this is a bug but when animations are enabled:

```
import am4themes_animated from '@amcharts/amcharts4/themes/animated';
import am4themes_kelly from '@amcharts/amcharts4/themes/kelly';
am4core.useTheme(am4themes_animated);
am4core.useTheme(am4themes_kelly);
```

And there is only one Category axis (xAxis) data point:

```
      this.zone.runOutsideAngular(() => {
        const config = {
           // >>>>>>>>>>>> here, this.sessions is an array with only one entry.
          data: this.sessions.map((sess) => ({
            categoryId: sess.sessionId,
            notes: sess.notes,
            pain: sess.pain,
            wellness: sess.wellness,
          })),
        };

        const chart = am4core.createFromConfig(config, 'chartdiv', am4charts.XYChart) as am4charts.XYChart;
        chart.width = am4core.percent(100);
        chart.height = am4core.percent(100);

        const series = chart.series.getIndex(0) as am4charts.ColumnSeries;
        const columnTemplate = series.columns.template;

        const columnHoverState = columnTemplate.column.states.create('hover');
        columnHoverState.properties.cornerRadiusTopLeft = 10;
        columnHoverState.properties.cornerRadiusTopRight = 10;

        chart.padding(0, 0, 40, 0);
        this.chart = chart;
      });
```

Then I see the following:

![image](https://user-images.githubusercontent.com/13020457/43916193-7deb2dc0-9bd2-11e8-9e2e-6a0a8e538909.png)

**Comment by martynasma** [comment]
Nice. Looks kinda dope :)

Any chance you could post your full chart code so that we can set up a test?

This looks like only a fraction of it.

**Comment by codinronan** [comment]
@martynasma yep, confirmed I'm allowed to post it, give me a bit of time so I can give you a JSON blob in the right format to test with too :)

**Comment by martynasma** [comment]
If you could create a CodePen that would be even better ;)

**Comment by martynasma** [comment]
Hey, just following up. Have you had a chance to post your chart code and/or demo? We're having trouble reproducing this here, but got another report of similar behavior, so it's definitely a thing. Having a way to reproduce it would be tremendous help. Thanks!

**Comment by dswitzer** [comment]
I'm seeing the same behavior, even when animations are disabled. 

At least in my case, I've found that tweaking the `minGridDistance` can stop the behavior, but I suspect it could have side effects with other charts that have data (since I have to pick a value specific to avoid the chart from collapsing).

**Comment by martynasma** [comment]
Yup this does not seems to be related to animations.

The important thing is that we were able to reproduce this in "lab conditions". Hopefully we'll be able to provide a fix soon.

**Comment by Taylorsuk** [comment]
@martynasma working on creating the Codepen with @codinronan. We've found that it's impossible to reproduce in a local dev environment, it's only happening in Chrome and when there is one Bar chart displayed it seems. If there is multiple bars on the chart then it works as expected. Unsure if those points give you any leads.

**Comment by martynasma** [comment]
@Taylorsuk Thanks for the heads up.

It seems we were able to reproduce it and working on a fix. But if its not too much trouble, I'd still love for you to post your demo, so that we can run fix against your setup as well.
